src/inputs/Link/index.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import Link from './index';
+import { Icon } from '../../';
+
+export default {
+  title: 'inputs/Link',
+  component: Link,
+  parameters: {
+    componentSubtitle: 'Link component.',
+  },
+};
+
+export const Default = (): React.ReactElement => (
+  <Link href="#">Some text</Link>
+);
+
+export const withColor = (): React.ReactElement => (
+  <Link href="#" color="error">
+    Some text
+  </Link>
+);
+
+export const WithCustomSize = (): React.ReactElement => (
+  <Link href="#" size="lg">
+    Some text
+  </Link>
+);
+
+export const WithCustomChild = (): React.ReactElement => (
+  <Link href="#" size="lg">
+    <Icon type="alert" size="md" />
```

Comment:
The icon is a bit off. 
I think adding this it will work
```
display: flex;
align-items: center;
```

Reply:
ok, the idea is to show that it's possible to add a child different than simple text, but I'll add a container to align it. :smile: